src/index.html

```diff
@@ -9,6 +12,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="home_page_link" href="#home">
+        <img
+        class="logo_img--link-to-home-page"
```

Comment:
if the tag has a class with modificator, then the tag should initial class, for example:
`class="logotype logotype--small"`